From 60cd8228e79151fbb2fe574161da5743937a0122 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Mon, 13 Aug 2018 13:38:46 +0200
Subject: [PATCH 04/16] Revert "util: jsoncompat: Stub out virJSONInitialize
 when compiling without jansson"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 9e44c2db8ad94d3c20acc1d081538c280af198b4.

Jansson cannot parse QEMU's quirky JSON.
Revert back to yajl.

https://bugzilla.redhat.com/show_bug.cgi?id=1614569

Signed-off-by: Ján Tomko <jtomko@redhat.com>
Reviewed-by: Daniel P. Berrangé <berrange@redhat.com>
---
 src/util/virjsoncompat.c | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/src/util/virjsoncompat.c b/src/util/virjsoncompat.c
index ffbeb5488c..6c853e9bb5 100644
--- a/src/util/virjsoncompat.c
+++ b/src/util/virjsoncompat.c
@@ -271,15 +271,4 @@ json_true_impl(void)
     return json_true_ptr();
 }
 
-
-#else /* !WITH_JANSSON */
-
-
-int
-virJSONInitialize(void)
-{
-    return 0;
-}
-
-
-#endif /* !WITH_JANSSON */
+#endif /* WITH_JANSSON */
-- 
2.18.0

